narwhal/consensus/src/tests/randomized_tests.rs

```diff
@@ -497,12 +532,13 @@ fn generate_and_run_execution_plans(
         // Now create a new Bullshark engine
         let metrics = Arc::new(ConsensusMetrics::new(&Registry::new()));
         let mut state = ConsensusState::new(metrics.clone(), gc_depth);
+        const SUB_DAGS_PER_SCHEDULE: u64 = 5;
         let mut bullshark = Bullshark::new(
             committee.clone(),
             store.clone(),
-            latest_protocol_version(),
+            protocol_config.clone(),
```

Comment:
just curious, is it necessary to both take `protocol_config` by value and clone it here?

Reply:
good point, we could just take `protocol_config` by ref